app/lib/synthetic_branding_group_populator.rb

```diff
@@ -0,0 +1,107 @@
+##
+# SyntheticBrandingGroupPopulator: class to populate branding groups using the output of SyntheticStudyPopulator
+##
+
+class SyntheticBrandingGroupPopulator
+  BASE_BRANDING_GROUPS_CONFIG_PATH = Rails.root.join('db', 'seed', 'synthetic_branding_groups')
+
+  # populate all branding groups using synthetic studies
+  #
+  # @param user (User)                 => User account to own the branding groups
+  # @param overwrite_groups (Boolean)  => T/F for destroying/recreating any existing groups
+  # @param overwrite_studies (Boolean) => T/F for destroying/recreating any existing studies
+  def self.populate_all(user: User.first, overwrite_groups: true, overwrite_studies: false)
+    # populate all studies first to ensure they exist
+    SyntheticStudyPopulator.populate_all(overwrite: overwrite_studies)
+    synthetic_studies_ids = self.collect_synthetic_studies.pluck(:id)
+    # divide into two groups, one for human data, another for mouse
```

Comment:
There should be a field in the json seed files 'study_title_regex' that contains the filter for which studies are in the group, rather than hardcoded in the populator—I think that will make this code cleaner.  

That way this method can handle finding the config directories and parsing the config files, and then populate can take a user, the parsed config object, and the options, and populate will handle finding the study lists

Reply:
Makes sense.

Reply:
568c460, 7290786